### ISSUE TITLE
hicPlotDistVsCounts table of counts

### ISSUE THREAD

**Issue by apaytuvi** [maxplanck-ie/HiCExplorer#77]
Is it possible to get the data from `hicPlotDistVsCounts`? In a txt file, for instance.

Thanks.

**Comment by fidelram** [comment]
Should be easy to implement. I will take a look.

Meanwhile, you can look at this line:

https://github.com/maxplanck-ie/HiCExplorer/blob/master/hicexplorer/hicPlotDistVsCounts.py#L91

and save the variables using json, assuming that your goal is to use R, for example for plotting. 

The x values of the plot are:

`dist_list = np.sort(common_dist)
`
the y values are the mean of each distance key `diagonals_dict[chrom]` 

In other words, for distance 10kb in chromosome chr2 the mean is `np.mean(diagonals_dict[chrom][10000])`

**Comment by apaytuvi** [comment]
Thank you!

**Comment by fidelram** [comment]
I introduced significant changes to `hicPlotDistVsCounts` including the option to save the underlying data.

Now it is also possible to combine in one plot the lines from different matrices.

This is in the develop branch at the moment.
